period_search_optimization_simd/period_search/CpuInfoArm.cpp

```diff
@@ -71,9 +124,11 @@ void SetOptimizationStrategy(SIMDEnum useOptimization)
 {
 	switch (useOptimization)
 	{
+#if !(defined(__linux__) && (defined(__aarch64__) || defined(_M_ARM64)))
```

Comment:
never use asimd on 64bit linux? this looks wrong or at least not in the right place...